src/System.IO.FileSystem/tests/FileSystemTest.cs

```diff
@@ -49,5 +49,51 @@ public abstract partial class FileSystemTest : RemoteExecutorTestBase
 
             return success;
         });
+
+        /// <summary>
+        /// Runs the given command as sudo
+        /// </summary>
+        /// <param name="commandLine">The command line to run as sudo</param>
+        /// <returns> Returns 0 if the process exits</returns>
```

Comment:
Nit: it returns the exit code of the process... that'll be 0 if the process ran successfully, but it could be something else if the program exits with an error.